### PR TITLE
fix wms error format

### PR DESCRIPTION
**Pull request by michaelmattig**
- [ ] I added an entry to [`CHANGELOG.md`](CHANGELOG.md) if knowledge of this change could be valuable to users.

---

Here is a brief summary of what I did:

<TEXT>

### CODE REVIEW

**Comment by michaelmattig** [comment]
[Fix: cartopy WMS](https://trello.com/c/pImtg5PS/1163-fix-cartopy-wms)

File: services/src/handlers/wms.rs [hunk 1]
```diff
@@ -186,9 +184,7 @@ where
             </GetMap>
         </Request>
         <Exception>
-            <Format>XML</Format>
-            <Format>INIMAGE</Format>
-            <Format>BLANK</Format>
+            <Format>application/json</Format>
```
Comment (jdroenner) [inline services/src/handlers/wms.rs:187]: is this a valid option?

Comment (michaelmattig) [reply services/src/handlers/wms.rs:187]: actually no, the standard says

- XML (mandatory)
- INIMAGE (optional)
- BLANK (optional)

default value is also XML.

We could have our frontend specify our "vendor specific" application/json and implement XML additionally. Or, like before, just ignore the parameter and always output json regardless. But I guess one of the two ist has to be in order for the examples to work again.

Comment (jdroenner) [reply services/src/handlers/wms.rs:187]: i wonder what happens if we don't declare on of the official options. XML is mandatory...
We should add json but maybe add some minimal form of xml? Do you know what the minimal requirement for XML would be?

Comment (michaelmattig) [reply services/src/handlers/wms.rs]: I added XML exception format: c270e3e3bc4419071d65f1e39fb0d416b9546c01

**Review by jdroenner — APPROVED** [review]

**Comment by jdroenner** [comment]
cargo merge

**Comment by jdroenner** [comment]
bors merge

**Pull request merged**